docs/unified-test-documentation/dasharo-security/206-secure-boot.md

```diff
@@ -499,3 +499,914 @@ incorrect format.
+    [Generic test setup: firmware](../generic-test-setup.md#firmware).
+1. Proceed with the
+    [Generic test setup: OS installer](../generic-test-setup.md#os-installer).
+1. Proceed with the
+    [Generic test setup: OS installation](../generic-test-setup.md#os-installation).
+1. Proceed with the [DTS: Build image with UEFI Secure Boot
+    support](../../dasharo-tools-suite/documentation.md#build-image-with-uefi-secure-boot-support).
+
+**Test steps**
+
+1. Power on the DUT.
+1. While the DUT is booting, hold the `BIOS_SETUP_KEY` to enter the UEFI setup
+    menu.
+1. Enter the `Device Manager` menu using the arrow keys and Enter.
+1. Enter the `Secure Boot Configuration` submenu.
+1. Set the `Secure Boot Mode` field to `Custom Mode`.
+1. Erase Secure Boot keys select options in the given order: `Custom Secure Boot Options` ->
+    `DB Options` -> `Enroll Signature` -> `Erase all Secure Boot Keys`.
+1. Press `F10` to save changes.
+1. Press `ESC` until the setup menu.
+1. Select the `Reset` option.
+1. While the DUT is booting, hold the `BOOT_MENU_KEY` to enter the boot menu.
+1. Boot Dasharo Tools Suite from `USB Storage`.
+1. Wait until Dasharo Tools Suite enrolls keys and resets the DUT.
+1. While the DUT is booting, hold the `BIOS_SETUP_KEY` to enter the UEFI setup
+    menu.
+1. Enter the `Device Manager` menu using the arrow keys and Enter.
+1. Enter the `Secure Boot Configuration` submenu.
+1. Set the `Current Secure Boot State` field to `Enabled`.
+1. Press `F10` to save changes.
+1. Press `ESC` until the setup menu.
+1. Select the `Reset` option.
+1. Verify by booting signed Dasharo Tools Suite:
+    1. While the DUT is booting, hold the `BOOT_MENU_KEY` to enter the boot menu.
+    1. Boot Dasharo Tools Suite from `USB Storage`.
+1. Reboot the DUT.
+1. Verify by booting unsigned Ubuntu:
+    1. While the DUT is booting, hold the `BOOT_MENU_KEY` to enter the boot menu.
+    1. Boot `OPERATING_SYSTEM`.
+
+**Expected result**
+
+1. Signed Dasharo Tools Suite should boot.
+1. Unsigned Ubuntu should not boot.
+
+## SBO013.002 Check automatic certificate provisioning KEK certificate
+
+**Test description**
+
+This test verifies that the automatic certificate provisioning installs the
+expected KEK certificate. Before launching test, make sure that DTS with
+automatic certificate provisioning is attached and can be booted on DUT.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo.
+1. `OPERATING_SYSTEM` = Dasharo Tools Suite.
+1. Additional `USB storage`for keeping Dasharo Tools Suite.
+1. Dasharo Tools Suite with UEFI secure boot support.
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../generic-test-setup.md#firmware).
+1. Proceed with the
+    [Generic test setup: OS installer](../generic-test-setup.md#os-installer).
+1. Proceed with the
+    [Generic test setup: OS installation](../generic-test-setup.md#os-installation).
+1. Proceed with the [DTS: Build image with UEFI Secure Boot
```

Comment:
@TomaszAIR I suppose DTS has functionality of automatic certificate provisioning,  on current release, hasn't it?

Reply:
https://github.com/Dasharo/docs/pull/749#discussion_r1523316578